tools/hilow.wdl

```diff
@@ -0,0 +1,468 @@
+version 1.1
+
+task active_regions_merge {
+    meta {
+        description: "Merge active regions"
```

Comment:
Can you add a `help` that summarizes the Bash below? I could dig in to follow along, but a plaintext description would be easier, especially as the literal Bash won't be present in the `sprocket doc` output.

Reply:
Yes, but I'll have to dig in to the bash a little more. This is just a copy/paste of HiLOW code.